compile.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+RED=$(tput setaf 1)
+RESET=$(tput sgr0)
+
+# Run with `bazel` command as param (build|test)
+if [[ $1 != "build" && $1 != "test" ]]; then
+  echo "$RED""Run script with 'build' or 'test' as param$RESET"
+  exit 1
+fi
+
+# Builds all targets and runs all tests
+PROTO_OPTIONS="--proto_compiler //external:proto_compiler --proto_toolchain_for_java //external:proto_java_toolchain"
```

Comment:
Do we still need these here if they're in `bazelrc`?
Also, when should this script be run? Please document at the top.

Reply:
@oferb I think we don't
You should run to regression-test all targets (build or test everything) -- will document